tests/unit/lms/services/dashboard_test.py

```diff
@@ -217,46 +217,53 @@ def test_get_request_admin_organizations_for_staff(
 
         assert svc.get_request_admin_organizations(pyramid_request) == [organization]
 
+    @pytest.mark.parametrize("rosters_enabled", [True, False])
+    @pytest.mark.parametrize("roster_available", [True, False])
```

Comment:
Combined a couple of test into this one, adding all the combinations of enabled/disabled with the data available/missing.